docs/modules/device-lab-management/pages/android-devices/prepare-android-device.adoc

```diff
@@ -0,0 +1,231 @@
+= Prepare an Android device for hosting
+:navtitle: Prepare an Android device
+
+Learn how to prepare an Android device to be utilized with Kobiton.
+
+Different models of devices and versions of Android may have different layout for settings and options. Refer to the device's manual if you cannot find the options mentioned in this document. In this guide, the screenshots from _Samsung Galaxy_,  _Google Pixel_, and _Xiaomi Redmi_ devices are used as example.
+
+== Before you start
+
+You'll need the following:
+
+* A supported xref:supported-platforms:mobile-operating-systems-and-devices.adoc[Android device model].
+
+* Additional requirements for Xiaomi models:
+
+** A logged in **MI account**. You can create a MI account on Xiaomi’s https://account.xiaomi.com/[account page].
+
+** A **SIM card** inserted. It does not need to be a working SIM card.
+
+== Factory reset the device (optional)
+
+[NOTE]
+This step is highly recommended for devices that were hosted by Legacy Kobiton or Legacy Gigafox.
+
+[CAUTION]
+This action will remove all data on the device, back up the device before proceeding.
+
+Refer to the device model's manual for instructions to factory reset the device.
+
+== Enable USB debugging (required)
+
+[IMPORTANT]
+====
+
+This step is required for Kobiton software to control the device.
+
+====
+
+From the Home screen, swipe up to show all apps. Open **Settings**.
+
+Follow the appropriate steps depending on whether the device is made by Xiaomi or not.
+
+[tabs]
+====
+
+Most devices::
++
+--
+
+Scroll down and select **About phone**. For some models, such as Google Pixel, this option is under **System**.
+
+Select **Software information**.
+
+image:device-lab-management:device-lab-management-android-prepare-usb-debugging-settings.png[width=240, alt="The Settings app in the home screen"] -> image:device-lab-management:device-lab-management-android-prepare-usb-debugging-about-phone.png[width=240, alt="About Phone in Settings menu"] -> image:device-lab-management:device-lab-management-android-prepare-usb-debugging-software-information.png[width=240, alt="Software Information section inside About Phone"]
+
+Tap 7 times on **Build number**. A popup message will appear when you are close to enabling the mode. If you have a lock screen set, you’ll be asked to enter your security credentials.

```

Comment:
In such cases I always refer to Google Developer Documentation Style Guide: https://developers.google.com/style/word-list#letter-p

According to this style guide, you should not use the word popup or pop-up in the first place. Instead, you should use Menu or Dialog
![110](https://github.com/kobiton/docs/assets/20087517/19a55dc3-b687-41c2-9fb2-745831b85149)

Reply:
We are not Google, so for us, instead of going back changing each and every instance of the word, it is simply cost efficient to decided on either pop-up or popup. 

I recommend popup since it is the word structure that the modern reader is most familiar with.

Reply:
Such a style guide is overshadowed by a million other articles on the internet that were and are used to train GPT. Since GPT acts in a way that is acted in, or used, the most, such a specific style guide material is basically non-existent in any form of GPT.

Reply:
Let's use `popup` then.

Reply:
replace to popup on all pages in the android-devices and ios-devices pages.